src/GHC.hs

```diff
@@ -162,6 +163,7 @@ stage0Packages = do
              , unlit                       ]
           ++ [ terminfo | not win, not ios, not cross ]
           ++ [ touchy   | win              ]
+          ++ [ timeout ]
```

Comment:
Does the Make build system always build `timeout` or only when running tests?

Reply:
I just trying to make it buildable.... Indeed I'm not very sure what I was doing...

Reply:
It's only used for the tests indeed, we might get away with simply throwing the rules for the timeout program in `Rules/Test.hs` if they're not too complicated (the non-windows case is trivial, but the windows case a little less so).

Reply:
This is fine for debugging to cause `timeout` to be built by default. But I agree with @alpmestan that when the build works, we'll just need to add `timeout` to the list of all packages and `need` it from the test rule.